cli/tests/integrations/test_package.py

```diff
@@ -62,10 +66,13 @@ def test_update_with_validation():
         ['dcos', 'package', 'update', '--validate'])
 
     assert returncode == 0
-    assert b'source' in stdout
-    assert b'Validating package definitions...' in stdout
-    assert b'OK' in stdout
     assert stderr == b''
+    if os.environ.get(constants.COSMOS_URL_ENV) is None:
+        assert b'source' in stdout
+        assert b'Validating package definitions...' in stdout
+        assert b'OK' in stdout
+    else:
+        assert stdout == b'This command is being deprecated\n'
```

Comment:
Sorry should have mentioned this before. "This command is deprecated\n". From their point of view it already happened. :)